frontend/src/metabase/querying/notebook/components/Notebook/context.tsx

```diff
@@ -0,0 +1,32 @@
+import { type PropsWithChildren, createContext, useContext } from "react";
+
+import type { DataPickerValue } from "metabase/common/components/DataPicker";
+
+type NotebookContextType = {
+  modelsFilterList: DataPickerValue["model"][];
+};
+
+export const NotebookContext = createContext<NotebookContextType | undefined>(
+  undefined,
+);
+
+export const NotebookProvider = ({
```

Comment:
Can't this be `NotebookProvider = NotebookContext.Provider`?

Reply:
We would have to initialize the NotebookProvider with 

<NotebookProvider value={{ modelsFilterList: [...] }}>
    {... whatever ...}
</NotebookProvider>


like we do here, but for all of the instances of the NotebookProvider. I would prefer to keep the props the same as other components